### PR TITLE
Fix failing test

### PR DESCRIPTION
**Pull request by DillonOLeary**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Fix a failing test. The are a number of listing changes as well that have no impact on the code. The only change that matters is line 160 and 169, using `console.error` instead of `console.log`.

## Related Tickets & Documents

No related tickets

## QA Instructions, Screenshots, Recordings

`yarn test` works now

## Added/updated tests?
_We are aiming for 100% statement coverage._

- [ ] Yes
- [x] No, and this is why: This is a test fix already
- [ ] I need help with writing tests

## What gif best describes this PR or how it makes you feel?

animal crossing gif

### CODE REVIEW

File: __tests__/src/components/serviceRequest/ServiceRequestDialog.test.tsx [hunk 1]
```diff
@@ -1,176 +1,171 @@
...
-    clientService.newRequest = jest.fn().mockImplementation(async () => {
-      return new Promise((undefined, r) => reject = r)
-    })
-    const testError = 'Test fetch failed'
-    console.log = jest.fn()
-    setup(true)
+    let reject;
+    clientService.newRequest = jest.fn().mockImplementation(async () => new Promise((undefined, r) => reject = r));
+    const testError = 'Test fetch failed';
+    console.error = jest.fn();
```
Comment (DillonOLeary) [inline __tests__/src/components/serviceRequest/ServiceRequestDialog.test.tsx:160]: This and line 169 are the only changes that matter

File: __tests__/src/components/serviceRequest/ServiceRequestDialog.test.tsx [hunk 2]
```diff
@@ -1,176 +1,171 @@
...
+    await waitFor(() => expect(clientService.newRequest).toHaveBeenCalledTimes(1));
 
     //* Assert
-    await waitFor(() => reject(new Error(testError)))
-    expect(console.log).toHaveBeenCalledWith(testError)
-  })
-})
\ No newline at end of file
+    await waitFor(() => reject(new Error(testError)));
+    expect(console.error).toHaveBeenCalledWith(testError);
```
Comment (DillonOLeary) [inline __tests__/src/components/serviceRequest/ServiceRequestDialog.test.tsx:169]: This is the other change

**Comment by DillonOLeary** [comment]
looks like [pr 36](https://github.com/digitalaidseattle/seattle-humane-app/pull/36) fixes the same issue, I'll close this once that is merged

**Review by tristan-88 — APPROVED** [review]
Looks good

**Pull request merged**